javascript/index.js

```diff
@@ -14,52 +14,84 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+    // let displayTime = `${printMinutes()} ${printSeconds()}`;
+    printMinutes()
+    printSeconds()
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+  minUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  //change the DOM elements that display seconds 
+  //access the OL
+  const SplitTimeCollection = document.getElements
+  
+  secDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[0]
+  secUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[1]
 }
-
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
 }
 
 function printSplit() {
-  // ... your code goes here
+  let fetchSplit = chronometer.splitClick();
+  let newSplitsLi = document.createElement("li")
+   splits.appendChild(newSplitsLi).innerHTML = fetchSplit;
 }
 
 function clearSplits() {
```

Comment:
This function needs to be done so that the reset button can be used on your chronometer 😄